src/main/java/nl/ordina/jobcrawler/scrapers/YachtVacancyScraper.java

```diff
@@ -36,26 +34,26 @@ public YachtVacancyScraper(ConnectionDocumentService connectionDocumentService)
         //  Returns a List with VacancyURLs
         List<VacancyURLs> vacancyURLs = new ArrayList<>();
         int totalNumberOfPages = 1;
-        for (int pageNumber = 1; pageNumber <= totalNumberOfPages; pageNumber++) {
+        for(int pageNumber = 1; pageNumber <= totalNumberOfPages; pageNumber++) {
```

Comment:
Maybe we should all use the same code formatter for this project. I've noticed that @mostafa-221 inserts a lot of the spaces that are removed here.

Reply:
ok